python/perf-kernels/tools/tune_gemm/utils/utils.py

```diff
@@ -93,15 +94,15 @@ def patch_triton_compiler():
     if not triton_location_str:
         print("triton source not found from pip show triton")
 
-    triton_dir = triton_location_str[0].split()[-1].decode('utf-8')
+    triton_dir = Path(triton_location_str[0].split()[-1].decode('utf-8'))
 
-    jit_filename = os.path.join(triton_dir, "triton/runtime", "jit.py")
+    jit_filename = triton_dir / "triton" / "runtime" / "jit.py"
```

Comment:
I'm not familiar with such python hacks (or it's very common). Does this require a minimum python version?

Reply:
this is just [`pathlib`](https://docs.python.org/3/library/pathlib.html); minimum verison 3.4 :)